src/charts/progress/progress-chart.directive.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.charts.directive:pfProgressChart
+ *
+ * @description
+ *   Directive for rendering a progress chart
+ *
+ * @param {object} chartData the data to be shown in the progress chart<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.used   - number representing the amount used
+ * <li>.total  - number representing the total amount
+ * </ul>
+ *
+ * @param {object} config the chart configuration options<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.title  - the chart title
+ * <li>.units  - units to be displayed on the chart
+ * </ul>
```

Comment:
I think a config object is overkill here. Can we just use title and units parameters?